### PR TITLE
Adding new references

### PR DESCRIPTION
**Pull request by csinva**
Added new packages, general papers, and some papers on limitations of interpretable ML

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +138,9 @@ If you want to contribute to this list (*and please do!*) read over the [contrib
 * [A Survey Of Methods For Explaining Black Box Models](https://arxiv.org/pdf/1802.01933.pdf)
 * [A Marauder’s Map of Security and Privacy in Machine Learning](https://arxiv.org/pdf/1811.01134.pdf)
 * [Challenges for Transparency](https://arxiv.org/pdf/1708.01870.pdf)
-* [Explaining Explanations: An Approach to Evaluating Interpretability of Machine Learning](https://arxiv.org/pdf/1806.00069.pdf)
+* [Explaining Explanations: An Overview of Interpretability of Machine Learning](https://arxiv.org/pdf/1806.00069.pdf)
```
Comment (navdeep-G) [inline README.md]: Nice catch.

File: README.md [hunk 2]
```diff
@@ -59,6 +61,7 @@ If you want to contribute to this list (*and please do!*) read over the [contrib
 * [fairml](https://github.com/adebayoj/fairml)
 * [fairness](https://github.com/algofairness/fairness-comparison)
 * [foolbox](https://github.com/bethgelab/foolbox)
+* [iNNvestigate neural nets](https://arxiv.org/abs/1808.04260)
```
Comment (navdeep-G) [inline README.md]: This seems to be a paper link. Is there a link to an actual github project?

File: README.md [hunk 3]
```diff
@@ -75,6 +78,7 @@ If you want to contribute to this list (*and please do!*) read over the [contrib
 * [tensorflow/lucid](https://github.com/tensorflow/lucid)
 * [tensorflow/model-analysis](https://github.com/tensorflow/model-analysis)
 * [tensorflow/privacy](https://github.com/tensorflow/privacy)
+* [tensorfuzz](https://arxiv.org/abs/1807.10875)
```
Comment (navdeep-G) [inline README.md]: This seems to be a paper link. Is there a link to an actual github project?

File: README.md [hunk 4]
```diff
@@ -90,6 +94,7 @@ If you want to contribute to this list (*and please do!*) read over the [contrib
 * [featureImportance](https://github.com/giuseppec/featureImportance)
 * [forestmodel](https://cran.r-project.org/web/packages/forestmodel/index.html)
 * [fscaret](https://cran.r-project.org/web/packages/fscaret/)
+* [glm](https://www.rdocumentation.org/packages/stats/versions/3.6.0/topics/glm)
```
Comment (navdeep-G) [inline README.md]: Is there a github link or a cran.r-project.org link?

File: README.md (hunk 2 above)
Comment (csinva) [reply README.md]: Yes, sorry about that! Updated with github link.

File: README.md (hunk 3 above)
Comment (csinva) [reply README.md]: Yes, sorry about that! Updated with github link.

File: README.md (hunk 4 above)
Comment (csinva) [reply README.md]: Yes, sorry about that! Updated with github link.

**Comment by csinva** [comment]
updated arXiv links with github links

**Comment by jphall663** [comment]
@csinva glm should probably go in interpretable model section at bottom. BUT ... I feel like we've run you around a lot ... so I can merge this and fix myself? We do sincerely appreciate your contributions.

Also, I'm not sure how I feel about the new paper section. I've purposely had this repo focus on code and I think the reception has been good. For the paper links I might suggest this repo instead, which seems to be more focused on papers: https://github.com/lopusz/awesome-interpretable-machine-learning

@navdeep-G please don't merge yet.

**Comment by jphall663** [comment]
@csinva I am interested in the general papers you are proposing - I think we do probably want those.

**Comment by csinva** [comment]
Sure thing, feel free to do whatever you think is best and thanks for the pointer to the other repo!

**Comment by navdeep-G** [comment]
@jphall663 Thx for the review.

@csinva Thank you for the contribution!

**Pull request merged**